CodeStarterCamp_Week3/Step1.swift

```diff
@@ -5,10 +5,22 @@
 //  Created by karen on 2023/03/13.
 //
 
-let misterLee = Person(money: 1000)
-let missKim = Person(money: 1000)
-var yagomMenu: Dictionary<Coffee, Int> = [Coffee.moca: 5000, Coffee.iceMoca: 5500, Coffee.americano: 3000, Coffee.iceAmericano: 3500, Coffee.caramel: 4500, Coffee.iceCaramel: 5000, Coffee.espresso: 2500, Coffee.cappuccino: 4000, Coffee.hazelnut: 4000, Coffee.latte: 4000, Coffee.iceLatte: 4500]
-var yagombucks = CoffeeShop(totalSales:0, barista: misterLee, menu: yagomMenu)
+let misterLee = Person(money: 1000, name: "미스터 뤼~")
+let missKim = Person(money: 10000, name: "미스 김")
+var yagomMenu: Dictionary<Coffee, Int> = [
+    Coffee.moca: 5000,
+    Coffee.iceMoca: 5500,
+    Coffee.americano: 3000,
+    Coffee.iceAmericano: 3500,
+    Coffee.caramel: 4500,
+    Coffee.iceCaramel: 5000,
+    Coffee.espresso: 2500,
+    Coffee.cappuccino: 4000,
+    Coffee.hazelnut: 4000,
+    Coffee.latte: 4000,
+    Coffee.iceLatte: 4500
+]
```

Comment:
엄청나게 다양한 메뉴! 😲
타입 선언에서 Key의 타입을 지정해 주었기 때문에, 배열 안에서는 타입을 생략 할 수 있습니다! 😄
```suggestion
var yagomMenu: Dictionary<Coffee, Int> = [
   .moca: 5000,
   .iceMoca: 5500,
   .americano: 3000,
   .iceAmericano: 3500,
   .caramel: 4500,
   .iceCaramel: 5000,
   .espresso: 2500,
   .cappuccino: 4000,
   .hazelnut: 4000,
   .latte: 4000,
   .iceLatte: 4500
]
```

Reply:
오옷 이런 방법도 있군요. 해당 내용으로 수정 완료했습니다.